README.md

```diff
@@ -6,35 +6,43 @@ This repository applies [BERT](https://github.com/google-research/bert) to named
 
 ## Installation
 
-(will make available via `pip install` and `conda install` when project is closer to completion - hit the Watch button if you'd like to be updated on progress!)
+**NOTE**: Since there are quite a few requirements, creating a virtual environment for this repo is strongly recommended.
+
+**Instructions**: Use `conda` to create the environment. Currently environment creation via pip is not supported.
+
+To create the environment with `conda`, all you need to do is navigate to the `Multilingual_NER` directory and run the following in the command line:
+
+`conda env create -f environment.yml`
+
+You can verify that the required packages installed correctly by activating the environment (`conda activate m_ner`) and running `conda list`.
+
+Partial list of dependencies:  
+
+- pytorch-transformers 1.1.0+, torch 1.2.0+
+- spaCy (2.1.0+)
+- TensorFlow 1.14.0+, keras 2.2.4+ (only used for utility functions - will try to eliminate this dependency down the line)
+- numpy 1.15.4+, pandas 0.25.0+
+- pre-commit 1.18.0+
+- streamlit 0.4.0+
+- (Deployment dependencies - TBD)
+- Miscellaneous minor packages: pathlib, seqeval, tqdm, yaml
 
 ## Usage
 
 `python main.py` to train an English model and save per-epoch checkpoints.
 Add `-lang ru` to train a Russian model.
 
-Once you have a usable model checkpoint, open `python/demo.py` and set the path to the 
-model checkpoint you'd like to use and the demo's default language. Then run `streamlit run demo.py`, which opens a web app you can use to test the model's predictions interactively.
-
-## Requirements
-
--   **NOTE**: Since there are quite a few requirements, we recommend creating a virtual environment when using this package. (use `requirements.txt` when creating environment from scratch)
--   pytorch-transformers 1.1.0+, torch 1.2.0+
--   spaCy (2.1.0+)
--   TensorFlow 1.14.0+, keras 2.2.4+ (only used for utility functions - will try to eliminate this dependency down the line)
--   numpy 1.15.4+, pandas 0.25.0+
--   pre-commit 1.18.0+
--   streamlit 0.4.0+
--   (Deployment dependencies - TBD)
--   Miscellaneous minor packages: pathlib, seqeval, tqdm, yaml
+Once you have a usable model checkpoint, open `python/demo.py` and set the path to the model checkpoint you'd like to use and the demo's default language. Then run `streamlit run demo.py en`, which opens a web app you can use to test the model's predictions interactively. For the Russian demo, change `en` to `ru`.
```

Comment:
the path instructions here are no longer relevant based on how you have now set this up (which is great, btw). The demo works great!

Reply:
Very true, I'll amend this! And thanks! :)